File: app/src/main/java/com/example/java/kotlin/title/TitleManager.java
package com.example.java.kotlin.title;

public class TitleManager {

    private Title currentTitle;

    public void init() {
        currentTitle = Title.JAVA;
    }

    public void updateTitle() {
        switch (currentTitle) {
            case JAVA:
                currentTitle = Title.KOTLIN;
                break;
            case KOTLIN:
                currentTitle = Title.JAVA;
                break;
        }
    }

    public String getCurrentTitle() {
        return currentTitle.getTitle();
    }
}

File: app/src/main/java/com/example/java/kotlin/kotlin_code/main_activity/title/TitleManager.kt
package com.example.java.kotlin.kotlin_code.main_activity.title

class TitleManager {

    var currentTitle: Title

    init {
        currentTitle = Title.JAVA
    }

    fun updateTitle() {
        currentTitle = when (currentTitle) {
            Title.JAVA -> Title.KOTLIN
            Title.KOTLIN -> Title.JAVA
        }
    }

}